js/index.js

```diff
@@ -67,20 +67,23 @@ ctaText.append(ctaTextArray[1]);
 ctaText.append(lineBreak2);
 ctaText.append(ctaTextArray[2]);
 
-// let h1Arr = siteContent['cta']['h1'].split(' '); //apply split method on h1 string to create an array of strings stored in variable h1Arr
-// h1Arr.splice(1, 0, `<br>`); // use splice() on array to insert <br> at 1st index and remove no items
-// h1Arr.splice(3, 0, `<br>`); // use splice() on array to insert <br> at 3rd index and remove no items
-// let newH1Str = h1Arr.join(''); //join() joins the elements of h1Arr into a string newH1Stri
 
 // insert button text
 const buttonText = document.querySelector('button');
 buttonText.textContent = siteContent["cta"]["button"];
 
 //insert main content
-const mainHeaders = document.querySelectorAll('.main-content h4'); 
+let mainHeaders = document.querySelectorAll('h4'); 
```

Comment:
Definitely prefer grabbing all the elements so I can loop over them!  Only thing to keep in mind with this kind of approach is if anyone else ever touches your code, they could mess it up very quickly adding another h4!